GeneratorInterface/Hydjet2Interface/src/Hydjet2Hadronizer.cc

```diff
@@ -462,6 +459,9 @@ bool Hydjet2Hadronizer::get_particles(HepMC::GenEvent *evt) {
     }
     ihy++;
   }
+  if (sub_vertices)
```

Comment:
There is no need to check that the pointer is non-null before calling delete. Internally delete does such a check.